### DISCUSSION TITLE
WebdriverIO > Slack Reporter

### DISCUSSION THREAD

**Discussion by semeale** [webdriverio/webdriverio#8437] [Q&A]
Hello everyone, I am currently working on WebdriverIO Slack reporter and it sends notifications for each test(spec) and it makes the channel very noisy. I was wondering if there’s a way to send notifications on the suite level when it runs the entire suite and sends only one notification with suite results. Currently, when I am running the suite it sends notifications for each test inside of the suite. Thank you all in advance. My test folder looks like this:
Project folder >Test spec 1(1test), Test spec 2(1 test), Test spec 3(1 test)

Thank you all in advance.

**Comment by christian-bromann** [comment]
Have you tried different reporter events, e.g. [`onSuiteEnd`](https://github.com/webdriverio/webdriverio/tree/main/packages/wdio-reporter#onsuiteend)?
